### PR TITLE
[Sprites] guitar back and some resprites

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

welding masks icon repsrites
acoustic guitar backpack sprite
health analyzer icon resprite
fixed guitar inhand sprites

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: lzk
- tweak: Changed sprites

### CODE REVIEW

**Comment by OctoRocket** [comment]
It is missing screenshots

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@
         - HolofanProjector
         - Multitool
         - AppraisalTool
+        - TrayScanner
+        - GasAnalyzer
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:33]: These are components

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by lzk228** [comment]
shit... branches somehow merged together

**Review by AJCM-git — DISMISSED** [review]
Update the copyright in the ones you modified

**Comment by lzk228** [comment]
health analyzer meta was not updated in #16125, so i dont think that should be updated here

**Pull request merged**